### DISCUSSION TITLE
How can I search a word but excluding some directories using fzf-lua?

### DISCUSSION THREAD

**Discussion by xheisenbugx** [LazyVim/LazyVim#5080] [Q&A] [answered]
Hi everyone,

I’m looking for a way to achieve functionality similar to the [telescope-live-grep-args.nvim](https://github.com/nvim-telescope/telescope-live-grep-args.nvim) plugin, specifically the ability to use a glob filter to search for content while excluding certain files or folders from the results.

Does the default fzf-lua integration in LazyVim already support this, or would I need to create a custom command for it? If it requires customization, I’d appreciate guidance on how to set it up and bind it to a key.

Thanks in advance!

**Comment by minusfive** [comment, marked as answer]
This discussion should help you https://github.com/ibhagwan/fzf-lua/discussions/1501

**Reply by xheisenbugx** [reply]
Thanks, I added the next keymap and it works:

```
map("n", "<leader>/", LazyVim.pick("live_grep", { rg_glob = true }), { desc = "Live Grep with Dynamic Args" })
```
